UnitedIncome/components
# [APP-4064] InfoBox Updates/Tweaks based on Feedback

### Description
> Please provide a brief description of your changes below.

Got rid of onClick wrapper and infer the selectable css class. Also updated associated story and test.

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Checkout branch and ensure that the `InfoBox` and `InfoBoxFormattedRows` Atoms still load appropriately. More importantly, checkout the code updates 👨🏽‍💻

### Comments
> Any additional comments that will aid with the review process.

- [JIRA Ticket](https://unitedincome.atlassian.net/browse/APP-4064)
